doc/requirements.txt

```diff
@@ -5,19 +5,21 @@ jinja2==3.0.3
 sphinx==2.2.2
 m2r2==0.3.1
 mistune==0.8.4
-numba==0.53.1
+numba==0.56.4
 networkx==2.6.3
 numpy==1.19.2
 plotly==4.4.1
 protobuf<4.21.0  # remove when fix is released
 quantum-blackbird==0.4.0
+quantum-xir>=0.1.1
```

Comment:
Why has this been added?

Reply:
it was reported as missing in the readthedocs build, not sure why. I copied this from setup.py

Reply:
Seems a bit odd, but also fair to add it.